rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -727,15 +732,21 @@ def generate_wrapping_rosdoc2_sphinx_project_into_directory(
         index_rst_path = os.path.join(wrapped_sphinx_directory, 'index.rst')
         package = self.build_context.package
         if not os.path.isfile(index_rst_path):
-            # Generate a default index.rst
-            logger.info('Using a default index.rst.jinja')
-            template_path = resources.files(
-                'rosdoc2.verbs.build.builders').joinpath('index.rst.jinja')
-            template_jinja = template_path.read_text()
-
+            # Did the user provide index.rst.jinja?
+            user_jinja_path = os.path.join(wrapped_sphinx_directory, 'index.rst.jinja')
```

Comment:
Since we're using the more modern pathlib via resources below this could be symmetric and the `read_text()` method could be pulled outside the if block and avoid the `with open ... read()` Then the if block is just picking the template path and printing to say which it will use.